{"name": "test_get_user_by_gid", "status": "broken", "statusDetails": {"message": "TypeError: headers() got an unexpected keyword argument 'auth_token'", "trace": "token_client = <function token_client.<locals>.create_client at 0x7fe8e9a6ef70>\ntoken_storage = {}\n\n    @pytest.fixture(scope=\"session\")\n    def tenant_admin_token(token_client, token_storage):\n        client = token_client(get_config(\"BaseConfig\", \"base_url\"))\n>       status_code, response = client.create_tenant_admin_token()\n\n../../../conftest.py:89: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../clients/token_client.py:25: in create_tenant_admin_token\n    return self.create_token(create_tenant_admin_token, \"token_endpoint\")\n../clients/token_client.py:20: in create_token\n    status_code, response = self.request_processor(endpoint)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <products.reqres.clients.token_client.TokenClient object at 0x7fe8ea6056d0>\nendpoint = <products.reqres.endpoints.post_endpoint.PostEndpoint object at 0x7fe8ea605bb0>\nkwargs = {}, url = '/account/v1/token', http_method = 'POST'\nquery_params = None, auth_token = None, path_params = None\n\n    def request_processor(self, endpoint: IEndpointTemplate.__class__, **kwargs) -> (int, dict):\n        \"\"\"\n        This function processes the http request based on http methods\n        :param endpoint: it takes endpoint specifications which can be\n        provided by extending the \"IEndpointTemplate\" interface\n        :param kwargs: it takes keyword arguments required in special cases,\n        these are optional arguments\n        :return: it returns http status code and response\n        \"\"\"\n        url = endpoint.url()\n        http_method = endpoint.http_method()\n        query_params = endpoint.query_parameters()\n        auth_token = kwargs.get('auth_token')\n        path_params = endpoint.path_parameters(**kwargs) if 'user_id' in kwargs else None\n>       headers = endpoint.headers(auth_token=auth_token)\nE       TypeError: headers() got an unexpected keyword argument 'auth_token'\n\n../../../core/base/base_client.py:28: TypeError"}, "description": "Scenarios: Get Users", "start": 1718793328154, "stop": 1718793328154, "uuid": "7bf8bbdc-df8a-45aa-8e05-7ac7760ac0fc", "historyId": "af052d2a8c6240ecce7e812d5736bd94", "testCaseId": "af052d2a8c6240ecce7e812d5736bd94", "fullName": "products.reqres.tests.users_tests.TestUserModules#test_get_user_by_gid", "labels": [{"name": "story", "value": "Verify user modules"}, {"name": "epic", "value": "Users"}, {"name": "tag", "value": "dependency"}, {"name": "tag", "value": "reqres"}, {"name": "parentSuite", "value": "products.reqres.tests"}, {"name": "suite", "value": "users_tests"}, {"name": "subSuite", "value": "TestUserModules"}, {"name": "host", "value": "ldlkoushic-HP-Pavilion-Laptop-13-bb0078TU-Refurb"}, {"name": "thread", "value": "94653-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "products.reqres.tests.users_tests"}]}